internal/entities/models/models.go

```diff
@@ -0,0 +1,51 @@
+// Copyright 2024 Stacklok, Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package models contains domain models for entities
+package models
+
+import (
+	"github.com/google/uuid"
+	"github.com/stacklok/minder/internal/engine/entities"
+
+	"github.com/stacklok/minder/internal/db"
+	"github.com/stacklok/minder/internal/entities/properties"
+	minderv1 "github.com/stacklok/minder/pkg/api/protobuf/go/minder/v1"
+)
+
+type EntityInstance struct {
```

Comment:
This is just a convenience struct to avoid returning and carrying around both db.EntityInstance and properties when handling an entity with properties. We might want to use the newly introduced protobuf message instead.

Reply:
I agree that we should not use the database structures but should instead have either a middle-struct (as you did here) or even the protobuf. I leave that to your discretion.